docs/css/guides.css

```diff
@@ -0,0 +1,88 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+#guides-container {
+    max-width: 1440px;
+    margin: 0 auto;
+    padding-left: 20px;
+    padding-right: 20px;
+
+    @media (max-width: 991.98px) {
+        width: 100%;
+        padding-top: 46px;
+    }
+
+    @media (max-width: 767.98px) {
+        padding-top: 24px;
+    }
+}
+
+.guides-post-author-image {
+    height: 32px;
+    width: 32px;
+    border-radius: 100%;
+    vertical-align: middle;
+}
+
+.guides-post-column {
+    width: 100%;
+    color: #292828;
+}
+
+.guides-post-content {
+    width: 100%;
+    padding-top: 5px;
+    padding-left: 63px;
+    padding-right: 63px;
+    padding-bottom: 50px;
+
+    @media (max-width: 991.98px) {
+        padding-top: 30px;
+        padding-left: 33px;
+        padding-right: 33px;
+    }
+
+    @media (max-width: 767.98px) {
+        padding-top: 25px;
+        padding-left: 30px;
+        padding-right: 30px;
+    }
+}
+
+.guides-post-title-link {
+    font-family: 'IBM Plex Sans';
+    font-size: 30px;
+    color: #292828;
+    transition: color .2s;
+}
+
+.guides-post-paragraph {
+    text-align: justify;
+    overflow: hidden;
+    margin-top: 23px;
+    margin-bottom: 35px;
+}
+
+.guides-post-author-name {
```

Comment:
I think we can omit author info. Unlike blogs, we don't need to specify authors of guides.